lib/Net/Async/Github.pm

```diff
@@ -211,6 +213,39 @@ sub pr {
     )
 }
 
+=head2 prs
+
+Returns information of all PRs of given repository.
+
+Expects the following named parameters:
+
+=over 4
+
+=item * owner - which user or organisation owns this PR
+
+=item * repo - which repo it's for
+
+=back
+
+Return Ryu::Source object that resolves to the object of all PRs.
+
+=cut
+
+
+sub prs {
+    my ($self, %args) = @_;
+    die "needs $_" for grep !$args{$_}, qw(owner repo);
+    $self->validate_args(%args);
+    $self->api_get_list(
+        endpoint => 'pull_request',
+        endpoint_args => {
```

Comment:
just to undestand is this call 'pull_request' referring this hook https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#pull_request?  is it picking only open PR, how/where the open pull request alone picked up?

Reply:
https://docs.github.com/en/rest/reference/pulls#get-a-pull-request